### ISSUE TITLE
Resolution step deleting itself on pressing Ctrl + Backspace

### ISSUE THREAD

**Issue by Zeniker** [olivierkes/manuskript#375]
Steps taken (reproducible):

1. Open a project.
2. Enter in Plots.
3. Add a plot.
4. Add a resolution step.
5. On it's sumary, type anything.
6. Press Ctrl + Backspace and you will see the step you were editing suddenly being delete.

Version: 0.7.0

Software Versions in Use:
Python 3.6.5
PyQt 5.10.1
Qt 5.9.5

**Comment by din-din** [comment]
Especially annoying/interrupting if you come from another writing tool (like an Office type or even Docs) where you're used to use it to delete the whole word. 
Ctrl + Del works the same way, just with the word in front of the cursor. (This works in Manuskript as well)

**Comment by gedakc** [comment]
Thank you @Zeniker and @din-din for your input on Manuskript.

The behaviour of **CTRL+Backspace** removing a plot resolution step is a feature working as designed by the original author.  The corresponding key combination is **CTRL+Enter** to add a new resolution step.  These keyboard short cuts can be seen by hovering the mouse over the **+** and **-** icons under the resolution summary.

![manuskript-plot-resolution-step-hover-over-minus-icon](https://user-images.githubusercontent.com/10405019/48858081-c2ce6f00-ed77-11e8-9a73-c5a3e217b8fa.png)

Can you point out standards documentation (not just the key bindings for LibreOffice) that indicates that **CTRL+Backspace** should delete a word?

Is there an alternate combination of key bindings that would be preferable for adding and removing a plot resolution step?

Or should there be no key bindings for these functions?

**Comment by din-din** [comment]
CTRL + Backspace has been around for quite some time I think. Now, I can only speak from my Windows experience, but I can ask around at my university if it works the same on Mac and Linux. I do remember reading that Option + Delete does the same action on Mac, though.

It works from the same standard that CTRL + Left/Right Arrow Key lets you skip to either end of a word and CTRL + Del deletes the word on the right side of your cursor.

As far as I know these CTRL + whatever shortcuts work in most software types: browsers, sticky notes, all the office types I have worked with (LibreOffice, Microsoft, Google Services), I think it even worked in Visual Studio Code. It doesn't work in some software types, though, sometimes adding just a small square outline or nothing at all instead of deleting the word. The Text Editor in Windows doesn't support the shortcut for example.
(For example, just writing this reply, I used CTRL + Backspace a bunch of times to delete words, because it just goes much quicker than deleting a word letter by letter. And once a shortcut like that is in the memory, it's really hard to get it out again. Working with manuskript the other day I kept deleting half-written resolution steps without any way to recover them other than rolling back on an older save I had backed up somewhere. By now I've started to save a back-up copy after every sentence I write because I keep deleting whole steps instead of just one word.)

I'm not sure if that's what you're looking for, but take a look at this blog post: https://blogs.msdn.microsoft.com/oldnewthing/20071011-00/?p=24823/#5403635


As for the key binding for adding/removing resolution steps in general: I never use them. I'm just not adding or removing steps frequently enough to feel the need for a shortcut. 

What you could do if people really _need_ these shortcuts: make them configurable. So that people who don't feel the need for them can remove them in favour of a shortcut that's more comfortable for them to use.

**Comment by gedakc** [comment]
Thank you @din-din for the detailed feedback and link to a MS blog post.

I have since tested **CTRL+Backspace** in Firefox while editing this post and this key combination does delete the previous word.  I've also tested it in the Manuskript Editor pane and **CTRL+Backspace** deletes the previous word there as well.  Having two different key-binding behaviours is inconsistent and should be avoided.

I will investigate removing both the **CTRL+Enter** and **CTRL+Backspace** key bindings for adding and removing a plot resolution step respectively.
